components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -27,8 +27,8 @@
 
 static const CGFloat kNavigationBarDefaultHeight = 56;
 static const CGFloat kNavigationBarPadDefaultHeight = 64;
-static const UIEdgeInsets kTextInsets = {16, 16, 16, 16};
-static const UIEdgeInsets kTextPadInsets = {20, 16, 20, 16};
+static const UIEdgeInsets kTextInsets = {0, 16, 0, 16};
```

Comment:
Do these constants need to change?

Reply:
The insets are used to calculate the height of the titleView. However, because we are always centering the titleLabel within the titleView, it doesn't matter if titleView takes up the whole height of the MDCNavigationBar or not.

The reason it needs to change is that if the height of the navbar is smaller than topInset+titleLabelHeight+bottomInset, then titleLabel is centered in titleView, but not the navigation bar.

Reply:
Got it - thank you!